CHANGES.rst

```diff
@@ -2563,7 +2563,7 @@ astropy.wcs
 Other Changes and Additions
 ---------------------------
 
-
+- Updated required version of Cython to v0.29.13. [#9198]
```

Comment:
Maybe state the reason? Often times, when we bump minversion, some people are not too happy and demand to know why.

Reply:
Sounds good - does it look ok now?

Reply:
Yup, thanks!